src/server/api/index.js

```diff
@@ -2,9 +2,7 @@ require('dotenv').config();
 const express = require('express');
 const oauth = require('oauth-sign');
 const btoa = require('btoa');
-const json2xls = require('json2xls');

```

Comment:
@finnzink If you are no longer using json2xls, make sure to delete it from the package list as well.

yarn remove json2xls